### PR TITLE
chore(ci): turn off legacy plugin in check

### PR DESCRIPTION
**Pull request by hyrious**

### CODE REVIEW

**Review by BlackHole1 — APPROVED** [review]

File: .github/workflows/check-code.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@ jobs:
 
       - name: Build flat-web
         run: pnpm --filter flat-web build
+        env:
+            noLegacy: true
```
Comment (BlackHole1) [inline .github/workflows/check-code.yml:98]: miss `deploy`

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: .github/workflows/check-code.yml (hunk 1 above)
Comment (BlackHole1) [reply .github/workflows/check-code.yml:98]: 图片

Comment (hyrious) [reply .github/workflows/check-code.yml:98]: we don't turn off this plugin in deploy

Comment (hyrious) [reply .github/workflows/check-code.yml:98]: storybook actually does not build flat-web, so it is ok

**Review by crimx — APPROVED** [review]

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**